xunit-autolabeler-v2/ast_parser/python_bootstrap.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+import sys
+
+from python import invoker
+
+if len(sys.argv) != 2:
+    raise ValueError('Please specify exactly one [root] directory.')
+
+root_dir = sys.argv[1]
+output_file = os.path.join(root_dir, 'repo.json')
```

Comment:
I'm not sure if `repo.json` is the right filename. Can you suggest something else?

Reply:
How about `polyglot_snippet_data.json`?

(I'll update the name once we agree on one.)

Reply:
sounds good to me

Reply:
Thanks - updated.